### PR TITLE
added tests for radio list

### LINKED ISSUE #30: Backfill tests for Molecules/RadioList
Currently there is only a single default props "render-style" test

Using other component tests as a guide, write automated tests to cover important scenarios in component. Try to avoid granular assertions of the markup. Stay as high-level as possible to avoid test fragility.

### PR DESCRIPTION
**Pull request by SaidShah**
- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [x] Automated tests are passing
- [x] No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)
- [x] Localization: No hard-coded error messages in code files (_minimally_ in string constants)
- [x] New component and/ or properties? Make sure to update storybook

Closes #30

### CODE REVIEW

File: src/molecules/lists/radio-list.test.tsx [hunk 1]
```diff
@@ -1,6 +1,11 @@
 import React from "react";
 import { render } from "@testing-library/react";
-import { RadioList } from "./radio-list";
+import {
+    RadioList,
+    COMPONENT_CLASS,
```
Comment (jstevenson) [inline src/molecules/lists/radio-list.test.tsx:5]: Minor: alphabetize

File: src/molecules/lists/radio-list.test.tsx [hunk 2]
```diff
@@ -16,4 +21,32 @@ describe("RadioList", () => {
         // Assert
         expect(getByText(expected)).not.toBeNull();
     });
+
+    test("when items prop is empty, radio-list returns", () => {
```
Comment (jstevenson) [inline src/molecules/lists/radio-list.test.tsx:25]: Since `result` is null, shouldn't the name be something different than `radio-list returns`?

**Review by jstevenson — CHANGES_REQUESTED** [review]

**Review by jstevenson — APPROVED** [review]

**Review by brandongregoryscott — CHANGES_REQUESTED** [review]
Just two pieces of feedback to consider @SaidShah. At the very least I'd like to see the component class renamed so it isn't cluttering up potential imports. 

@wintondeshong do you have any opposition for pulling in that `jest-dom`package for additional matchers?

**Comment by SaidShah** [comment]
@brandongregoryscott - I made those requested changes if you would like to take a second look.

**Review by wintondeshong — APPROVED** [review]

**Pull request merged**